ruby_on_rails/app_initialisation.md

```diff
@@ -11,11 +11,16 @@
 * [Check if you are using the latest stable version of Rails](http://rubyonrails.org/) with `rails -v` and update it if you are not.
 You can do this with `gem update rails`. Beware of beta versions.
 
-* Start a new Rails project using `rails new [project-name] --database=postgresql --no-skip-test --skip-sprockets --webpack --skip_action_mailbox` where the `project-name` is exactly the one you chose before.
-You may want to choose a different database from Postgres, but most of the time that will be your choice.
+* Start a new Rails project using `rails new [project-name] --database=postgresql --skip-test --skip-action-mailbox` where the `project-name` is exactly the one you chose before.
```

Comment:
y was `--no-skip-test` used here before? I mean we manually delete the generated test stuff anyways because of RSpec

Reply:
I am good with it. @schmijos you did put some thoughts into that in the past. Any remark?

Reply:
I find it very weird that I added it here: https://github.com/renuo/applications-setup-guide/pull/174

I cannot remember why. Could it be that it comes with a Capybara and Selenium setup which we would have to do manually otherwise?

Generally: I'm fine with the suggested change.

Reply:
Yes, I guess, the reason is exactly what I thought. There are default configuration which we probably would like to keep:

![CleanShot 2022-03-28 at 10 31 50](https://user-images.githubusercontent.com/245443/160358496-cdd3524b-87be-4c5f-b35d-757372f89fd1.png)

Reply:
You even made a comment about it on the pull request: https://github.com/renuo/applications-setup-guide/pull/174/commits/359b3e2ce748c96ee60c456b623b4f7f06c17d49

> --no-skip-test should make clear that we want the test setup to be genereated even though we want to use RSpec later.